### PR TITLE
Rename Horror12 to 049Breath

### PR DESCRIPTION
**Pull request by MemeHoovy**
Horror12 is literally SCP-049 breathing, so I decided to rename it since it made more sense to have it like that (to me atleast).
I do understand if this doesn't get merged because of the low droning noise in the sound file.

### CODE REVIEW

**Review by ChronoQuote — CHANGES_REQUESTED** [review]
It looks like you've renamed Horror13.ogg through Horror16.ogg so as to "bump down" the numbers in the filenames by one, but this has consequences. Any time the game wants to play HorrorSFX(15), for example, it won't play the right sound anymore. Instead it'll play what was previously Horror16.ogg.

I see four options:
1. Keep the filenames changed, but also replace every instance of HorrorSFX(13) in the code with HorrorSFX(12), every instance of HorrorSFX(14) with HorrorSFX(13), and so on. Some parts of the code load the Horror sounds directly instead of using HorrorSFX, so those would also have to be replaced. High chance of missing one if you do this.
2. Keep the filenames changed, but also modify the loop in LoadAllSounds.bb that populates the HorrorSFX array, so that HorrorSFX(13) corresponds to the new Horror12.ogg, HorrorSFX(14) corresponds to the new Horror13.ogg, and so on. I don't think this is a good option either, as it introduces a confusing inconsistency between the array index and the filename, and you'd still have to replace all the parts of the code that load the sounds directly.
3. Revert the filenames to the way they were and accept the missing Horror12.ogg.
4. Revert all these changes (i.e. close the PR). After all, Horror12.ogg isn't the only horror sound specific to 049. Horror13.ogg is the sound of being kidnapped by 049.

**Comment by MemeHoovy** [comment]
> It looks like you've renamed Horror13.ogg through Horror16.ogg so as to "bump down" the numbers in the filenames by one, but this has consequences. Any time the game wants to play HorrorSFX(15), for example, it won't play the right sound anymore. Instead it'll play what was previously Horror16.ogg.
> 
> I see four options:
> 
> 1. Keep the filenames changed, but also replace every instance of HorrorSFX(13) in the code with HorrorSFX(12), every instance of HorrorSFX(14) with HorrorSFX(13), and so on. Some parts of the code load the Horror sounds directly instead of using HorrorSFX, so those would also have to be replaced. High chance of missing one if you do this.
> 2. Keep the filenames changed, but also modify the loop in LoadAllSounds.bb that populates the HorrorSFX array, so that HorrorSFX(13) corresponds to the new Horror12.ogg, HorrorSFX(14) corresponds to the new Horror13.ogg, and so on. I don't think this is a good option either, as it introduces a confusing inconsistency between the array index and the filename, and you'd still have to replace all the parts of the code that load the sounds directly.
> 3. Revert the filenames to the way they were and accept the missing Horror12.ogg.
> 4. Revert all these changes (i.e. close the PR). After all, Horror12.ogg isn't the only horror sound specific to 049. Horror13.ogg is the sound of being kidnapped by 049.

Whoops, I didn't know that. I'll just do the first one.

**Comment by ChronoQuote** [comment]
> Whoops, I didn't know that. I'll just do the first one.

Probably not worth the effort or the risk of a mistake to be honest.

File: Main.bb [hunk 1]
```diff
@@ -2151,7 +2151,7 @@ Function UpdateDoors()
 					EndIf
 					If d\AutoClose And RemoteDoorOn = True Then
 						If EntityDistance(Camera, d\obj) < 2.1 Then
-							If (Not Wearing714) Then PlaySound_Strict HorrorSFX(7)
+							If (Not Wearing714) Then PlaySound_Strict HorrorSFX(6)
```
Comment (ChronoQuote) [inline Main.bb:2154]: Horror7.ogg wasn't renamed to Horror6.ogg so this shouldn't have been changed.

Comment (MemeHoovy) [reply Main.bb]: Crap

**Review by ChronoQuote — CHANGES_REQUESTED** [review]
There's also a Horror16.ogg in UpdateEvents.bb. Then I think that's everything.

**Comment by ChronoQuote** [comment]
Just realized you'd also have to modify the second HorrorSFX loop in LoadAllSounds.bb so that Horror13.ogg and Horror14.ogg get loaded instead of Horror14.ogg and Horror15.ogg.
```Blitz3D
For i = 0 To 11
	HorrorSFX(i) = LoadSound_Strict("SFX\Horror\Horror" + i + ".ogg")
Next
For i = 14 To 15
	HorrorSFX(i) = LoadSound_Strict("SFX\Horror\Horror" + i + ".ogg")
Next
```

**Comment by MemeHoovy** [comment]
I had 

> Just realized you'd also have to modify the second HorrorSFX loop in LoadAllSounds.bb so that Horror13.ogg and Horror14.ogg get loaded instead of Horror14.ogg and Horror15.ogg.
> 
> ```blitzmax
> For i = 0 To 11
> 	HorrorSFX(i) = LoadSound_Strict("SFX\Horror\Horror" + i + ".ogg")
> Next
> For i = 14 To 15
> 	HorrorSFX(i) = LoadSound_Strict("SFX\Horror\Horror" + i + ".ogg")
> Next
> ```

I had a feeling you'd say something about that

**Pull request closed without merging**